### PR TITLE
Add notebook with usage tutorial.

### LINKED ISSUE #85: Add a notebook targeting the deployed library

### PR DESCRIPTION
**Pull request by cristianberneanu**
Partially addresses #85.

### CODE REVIEW

**Comment by edongashi** [comment]
Can you please reset all cell numbers and re-run everything in order? We want them to be sequentially numbered:

image

File: docs/tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+   "outputs": [],
+   "source": [
+    "%pip install syndiffix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll need a toy dataset to play with, so let's install the `scikit-learn` package in order to use one of their popular reference datasets:"
```
Comment (edongashi) [inline docs/tutorial.ipynb:34]: Hmm, `scikit-learn` is a child dependency of ours. Does it work to import it directly?

**Review by edongashi — APPROVED** [review]

File: docs/tutorial.ipynb (hunk 1 above)
Comment (edongashi) [reply docs/tutorial.ipynb:34]: I guess it's better to keep it explicit in any case.

Comment (cristianberneanu) [reply docs/tutorial.ipynb:34]: Yes, that was my thinking as well.

**Pull request merged**